pkg/transformer/tlp.go

```diff
@@ -160,3 +155,70 @@ func partition(expr ast.ExprNode) []ast.ExprNode {
 func RandTLPType() TLPType {
 	return TLPTypes[util.Rd(len(TLPTypes))]
 }
+
+func tryAggTransform(selectStmt *ast.SelectStmt, unionStmt *ast.UnionStmt) (ast.ResultSetNode, error) {
+	if selectStmt.Fields != nil && len(selectStmt.Fields.Fields) != 0 {
+		aggFns := make([]int, 0)
+		exprNames := make(map[string]bool)
+		selectFields := make([]*ast.SelectField, 0, len(selectStmt.Fields.Fields))
+		unionFields := make([]*ast.SelectField, 0, len(selectStmt.Fields.Fields))
+		for index, field := range selectStmt.Fields.Fields {
```

Comment:
so it's unable to deal with aggregation functions in HAVING clause such as `SELECT * FROM a JOIN b GROUP BY c HAVING max(a.d) > 100`?

Reply:
It's unnecessary to deal with aggregation functions in HAVING clause